src/main/java/techcourse/myblog/user/User.java

```diff
@@ -0,0 +1,69 @@
+package techcourse.myblog.user;
+
+import javax.persistence.*;
+
+@Entity
+public class User {
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
+
+	@Embedded
+	@AttributeOverride(name = "username", column = @Column(name = "username"))
+	@AttributeOverride(name = "email", column = @Column(name = "email"))
+	@AttributeOverride(name = "password", column = @Column(name = "password"))
+	@AttributeOverride(name = "githubUrl", column = @Column(name = "githubUrl"))
+	@AttributeOverride(name = "facebookUrl", column = @Column(name = "facebookUrl"))
+	private Information information;
+
+	private User() {
+	}
+
+	public User(final Information information) {
+		this.information = information;
+	}
+
+	public void editUser(Information information) {
+		this.information.update(information);
```

Comment:
VALUE OBJECT는 불변이 특징입니다.

```suggestion
		this.information = information;
```

Reply:
네! 수정했습니다~!